client/src/pages/teamselection/SelectableRidersTable.tsx

```diff
@@ -1,58 +1,87 @@
-import DataTable, { TableColumn } from 'react-data-table-component';
-import RiderLink from '../../components/shared/RiderLink';
-import { SelectableEnum } from '../../models/SelectableEnum';
-import { SelectableRider } from './Models/SelectableRider';
-import SelectableRiderFoldout from './SelectableRiderFoldOut';
+import DataTable, { TableColumn } from "react-data-table-component";
+import RiderLink from "../../components/shared/RiderLink";
+import { SelectableEnum } from "../../models/SelectableEnum";
+import { SelectableRider } from "./Models/SelectableRider";
+import SelectableRiderFoldout from "./SelectableRiderFoldOut";
 
 const conditionalRowStyles = [
     {
         when: (row: SelectableRider) => row.selectable !== SelectableEnum.Open,
         style: {
-            backgroundColor: 'red',
-            color: 'white',
+            backgroundColor: "#450a0a",
         },
     },
     {
-        when: (row: SelectableRider) => row.selectable === SelectableEnum.Selected,
+        when: (row: SelectableRider) =>
+            row.selectable === SelectableEnum.Selected,
         style: {
-            backgroundColor: 'yellow',
+            backgroundColor: "#475569",
```

Comment:
moet misschien uit een style object komen of zo.
Ook wel leuk als we weer roze geel rood kunnen introduceren als accenten tijdens de races. (wss niet gerelateerd aan dit)